cpplite/include/blob/blob_client.h

```diff
@@ -412,7 +412,7 @@ namespace azure { namespace storage_lite {
         /// </summary>
         /// <param name="account_name">The storage account name.</param>
         /// <param name="account_key">The storage account key.</param>
-	/// <param name="sas_token">A sas token for the container.</param>
+	    /// <param name="sas_token">A sas token for the container.</param>
```

Comment:
There are two files which just has spacing related changes, we can revert them back.

Reply:
is there a problem with fixing the spacing related changes?

Reply:
Just dint want to touch the files we are not making any code changes.